libr/core/cconfig.c

```diff
@@ -2273,6 +2273,9 @@ static bool cb_utf8(void *user, void *data) {
 	RCore *core = (RCore *) user;
 	RConfigNode *node = (RConfigNode *) data;
 	core->cons->use_utf8 = node->i_value;
+#if __WINDOWS__
+	r_cons_win_set_cp (core->cons->use_utf8);
```

Comment:
what about naming it r_cons_set_utf8(), so using the setter/getter pattern instead of setting the use_utf8 variable by hand